app/src/main/java/app/revanced/manager/ui/screen/PatcherScreen.kt

```diff
@@ -172,4 +177,6 @@ fun PatcherScreen(
             }
         }
     }
-}
\ No newline at end of file
+}
+
+// Create an uninstall dialogue
```

Comment:
What is this?

Reply:
I think I was going to create my own uninstall dialog but then I saw it had already been created